healthcheck/sqlhealthcheck.go

```diff
@@ -54,32 +61,89 @@ func ReadYamlFromFile(path string) Format {
 func RunHealthChecks(healthChecks Format, cxn *sql.DB) Format {
 
 	for _, healthCheck := range healthChecks.Tests {
-		fmt.Println(healthCheck.Query)
-		rows, _ := cxn.Query(healthCheck.Query)
-		var answer string
-		rows.Next()
-		rows.Scan(&answer)
-		fmt.Println(answer)
-		healthCheck.Passed = healthCheck.Expected == answer
-		healthCheck.Actual = answer
-
-		fmt.Printf("HEALTH CHECK: %s, Expected: %s, Found:%s\n", healthCheck.Title, healthCheck.Expected, answer)
-
+		healthCheck = RunHealthCheck(healthCheck, cxn)
 	}
 	return healthChecks
 }
 
 // EvaluateHealthChecks contains logic for handling the results of RunHealthChecks
-func EvaluateHealthChecks(healthChecks Format) {
-	var errors []SQLHealthCheck
+func EvaluateHealthChecks(healthChecks Format) (results []SQLHealthCheck, err error) {
+
+	for _, healthCheck := range healthChecks.Tests {
+		results = append(results, healthCheck)
+		hcErr := EvaluateHealthCheck(healthCheck)
+
+		if hcErr.Err != "" {
+			err = errors.New(hcErr.Err)
+		}
+
+		if hcErr.Exit {
+			return results, err
```

Comment:
Perhaps "break" here, which will use the return statement below, rather than having the return same statement in multiple locations.